mithril-common/Cargo.toml

```
@@ -33,8 +33,12 @@ kes-summed-ed25519 = { version = "0.2.1", features = [
     "serde_enabled",
     "sk_clone_enabled",
 ] }
+minicbor = { version = "0.19", features = ["std", "half", "derive"] }
 nom = "7.1.3"
-pallas-network = { version = "0.20.0", optional = true }
+pallas-addresses = "0.21.0"
+pallas-codec = "0.21.0"
+pallas-network = { version = "0.21.0", optional = true }
+pallas-primitives = "0.21.0"
```

We need to have `pallas` crates behind the `fs` feature:
```suggestion
pallas-addresses = { version = "0.21.0", optional = true }
pallas-codec = { version = "0.21.0", optional = true }
pallas-network = { version = "0.21.0", optional = true }
pallas-primitives = { version = "0.21.0", optional = true }
```
And the feature would be defined as (line 101):
```
fs = [
    "tokio/fs",
    "tokio/process",
    "pallas-network",
    "pallas-addresses",
    "pallas-primitives",
    "pallas-codec",
]
```

---

just to make it documented,

we got errors to compile to warm after these changes.

@jpraynaud suggested to create a conditional macro to fix the build errors, resulting as:

```

macro_rules! cfg_fs {
    ($($item:item)*) => {
        $(
            #[cfg(feature = "fs")]
            #[cfg_attr(docsrs, doc(cfg(feature = "fs")))]
            $item
        )*
    }
}
```
thanks